app/views/patients/index.csv.erb

```diff
@@ -0,0 +1,8 @@
+<%=
```

Comment:
I do agree that it's pretty silly to have a template that's just a single big code block, and wish there were a best-of-both-worlds solution...